cove_360/templates/cove_360/additional_checks.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+{% load i18n %}
+{% block content %}
+
+<div>
```

Comment:
In think this ```<div>``` is missing the closing ```</div>```.

Reply:
I've added it to our html validation check in  https://github.com/OpenDataServices/cove/pull/840/commits/b1d75acc9e8e5a3f6a5669b44d4b5cf91c8cedf2